database_cleanup/models/purge_menus.py

```diff
@@ -29,17 +29,25 @@ def find(self):
         """
         Search for models that cannot be instantiated.
         """
+        def model_deleted(model_name):
+            """Check wether model has been deleted."""
+            # Testing for not in self.env does not work in 8.0:
+            if model_name:
+                try:
+                    test_model = self.env[model_name]  # noqa: F841
+                except KeyError:
+                    return True
+            return False
+
         res = []
         for menu in self.env['ir.ui.menu'].with_context(active_test=False)\
                 .search([('action', '!=', False)]):
             if not menu.action:
                 continue
             if menu.action.type != 'ir.actions.act_window':
                 continue
-            if (menu.action.res_model and menu.action.res_model not in
-                self.env) or \
-                    (menu.action.src_model and menu.action.src_model not in
-                        self.env):
+            if model_deleted(menu.action.src_model) or \
+                    model_deleted(menu.action.res_model):
```

Comment:
8 doesn't support the `in` operator on envs, but the registry does, which is accessible via `env.registry`